garage/tf/algos/npo.py

```diff
@@ -33,14 +33,33 @@ class NPO(BatchPolopt):
     """
     Natural Policy Gradient Optimization.
 
-    Attributes:
-        name(str): The name of the algorithm.
+    Args:
+        pg_loss(enum): The type of loss functions to use.
```

Comment:
you can call out the specific enum type (PGLoss) rather than just a generic enum

Note that we have issue #435 open to remove this enum anyway. Perhaps now is a good time to just replace it with the string?